src/Pages/AnswerPage/index.jsx

```diff
@@ -1,4 +1,5 @@
 import { Link } from "react-router-dom";
+import Badge from "../../components/common/Badge";
```

Comment:
```suggestion
import Badge from "@components/common/Badge";
```
그러니까 이렇게 쓸수있는 것이죠........... 
정작 제가 수정을 안했네요 ㅋㅋㅋ

Reply:
와 절대 경로!!! 

안그래도 ../../../ 이거 몇번 넘어가는건가 실눈 뜨고 세고 있었는데 👍🏻

Reply:
한 가지 단점이 있다면 절대경로로 불러온 환경변수는 옵션을 보여주지 않는다는 점입니다.......😭
환경변수는 그냥 자동으로 불러온 다음에 나중에 경로명 변경하시는게 쓰실 때 편할 것 같아요!

Reply:
넵 참고 하겠습니다!